README.md

```diff
@@ -1,72 +1,244 @@
+## Table of contents
+- [What is RIMReP DMS?](#what-is-rimrep-dms)
+- [Discovering for datasets in RIMReP DMS](#discovering-for-datasets-in-rimrep-dms)
+  * [Discovering for datasets via STAC](#discovering-for-datasets-via-stac)
+- [Code snippets](#code-snippets)
+  * [Connecting to tabular dataset in S3 bucket](#connecting-to-tabular-dataset-in-s3-bucket)
+  * [Extracting tabular data from S3 bucket](#extracting-tabular-data-from-s3-bucket)
+  * [Extracting gridded data from S3 bucket](#extracting-gridded-data-from-s3-bucket)  
   
-The good news is that you will not need to go through every notebook checking the libraries you need. We are providing some files that will automate this process for you whether you use `R`, `Python`, or both.  
+## More information
+- [Using API to access RIMReP DMS datasets](docs/rimrep_api_access.md)  
+- [Running example notebooks in this repository](docs/running_notebooks.md)    
+  * [Setting up your machine](docs/running_notebooks.md/#setting-up-your-machine)  
+- [Description of example notebooks in repository](docs/repository_file_description.md/#description-of-example-notebooks-in-repository)  
+- [Description of scripts in repository](docs/repository_file_description.md/#description-of-scripts-in-repository)  
 
-**Note:** You will only need to complete the set up steps once per machine. This should be done prior to running the notebooks for the first time. Also note that if you plan to use notebooks in one language, either `R` or `Python`, there is no need to follow the set up steps for the programming language that you do NOT need.
+## What is RIMReP DMS?
+RIMReP DMS is an Open Geospatial Consortium (OGC) API service and analysis-ready, cloud-optimised (ARCO) repository for data and metadata relevant to the management of the Great Barrier Reef. RIMReP DMS offers services to allow the discovery of the data and the interaction with external RIMReP systems.  
   
-### `R` users
-If you are using the `R` notebooks, run the following two lines in the `RStudio` console:  
-```R
-  source("R_based_scripts/Installing_R_libraries.R")  
-  checking_libraries()
-```  
-The lines above will run a function that automatically checks if any `R` libraries used in this repository are not installed in your machine. If any libraries are missing, it will install them automatically. Bear in mind that these notebooks were developed in `R` version 4.3.1, so you may need to upgrade your `R` version if you encounter any problems during package installation.
+In simple terms, RIMReP DMS is a data portal that aims to be a "one-stop-shop" for all data related to the Great Barrier Reef World Heritage Area, which can be easily accessed by the Great Barrier Reef Marine Park Authority (GBRMPA) to support evidence-based management strategies. All datasets have a standard format regardless of their origin, which not only facilitates access to data, but also their analysis as it removes the need to quality control individual datasets.  

```

Comment:
> but also their analysis as it removes the need to quality control individual datasets.  
This is not true. Replace with: 
but also their analysis as it removes the need to understand and parse different data formats